namiorg/flex-project-template
# [wip] iframe route security update

### Notable Changes
- Removed `darwin` os requirement from `package.json` files (they were there due to generation on Mac device)
- Added updated Flex CRM panel configuration to pass Worker SID and email to iframe for Worker validation

### Manual Testing
- Complete the testing setup steps from [namiorg/helpline-hub PR14](https://github.com/namiorg/helpline-hub/pull/14)
- Login to your local version of the `helpline-hub` site and make sure there is a user with the same email address you are using for the Twilio production account. If there is no user with this email, add one. 
- Clone the `flex-project-template` repo and checkout the `hh-route-update` branch
- From the repo root directory, run `npm ci` with the Twilio production account active on your Twilio cli
- Login to the Twilio production account in your browser at `console.twilio.com`
- From the `plugin-flex-ts-template-v1/` repo subdirectory, run `npm start`
- You should see the CRM iframe as expected and be able to view/add/update connections as expected without any issues
- Login to your local version of the `helpline-hub` site and delete the user with the same email address you are using for the Twilio production account 
- Clear your browser cache and restart the local plugin. You should see a 401 page since your user was (correctly) unable to be matched on both the Twilio database and `helpline-hub` database.
